src/Playwright/Core/Request.cs

```diff
@@ -39,24 +40,23 @@ internal class Request : ChannelOwnerBase, IChannelOwner<Request>, IRequest
     {
         private readonly RequestChannel _channel;
         private readonly RequestInitializer _initializer;
-        private readonly RawHeaders _headers;
+        private readonly RawHeaders _provisionalHeaders;
```

Comment:
nit: keep the name as is, we will update its value along the way.

Reply:
its like upstream now.